backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -33,7 +34,32 @@ public async Task<ActionResult<RepositoryModel>> GetRepository(string org)
                 }
             }
 
-            return StatusCode(404);
+            return StatusCode(204);
+        }
+
+        [HttpGet]
+        [Route("designer/api/{org}/resources/repository/resourcelist")]
+        public async Task<ActionResult<List<ContentsResponse>>> GetRepositoryResourceList(string org)
+        {
+            List<ContentsResponse> resourceList = new List<ContentsResponse>();
```

Comment:
ContentsResponse I assume is a Gitea Specific object. We should not expose this externally. 
Can we map this to a simple version of resource?